examples/Huggingface_Transformers/Download_Transformer_models.py

```diff
@@ -5,25 +5,26 @@
 import torch
 from transformers import (AutoModelForSequenceClassification, AutoTokenizer, AutoModelForQuestionAnswering,
  AutoModelForTokenClassification, AutoConfig)
+from transformers import set_seed
 """ This function, save the checkpoint, config file along with tokenizer config and vocab files
     of a transformer model of your choice.
 """
 print('Transformers version',transformers.__version__)
-
-def transformers_model_dowloader(mode,pretrained_model_name,num_labels,do_lower_case,max_length,torchscript):
+set_seed(1)
```

Comment:
Thanks Geeta, without setting the seed in the Download script, it does not show consistent behavior/ reproducible results  between eager mode and torchscript, when we are not using fine-tuned models. I believe this can be result of using torch.jit.trace as it may introduce some randomness.